src/encoded/static/components/filegallery.js

```diff
@@ -779,15 +778,19 @@ class RawFileTable extends React.Component {
                     {!this.state.collapsed ?
                         <tbody>
                             {groupKeys.map((groupKey, j) => {
-                                // groupFiles is an array of files under a bioreplicate/library
+                                // groupFiles is an array of files under a bioreplicate/library.
+                                // Determine whether to draw a bottom border based on whether this
+                                // is the last group displayed (don't draw) or not (do draw).
                                 const groupFiles = grouped[groupKey];
-                                const bottomClass = j < (groupKeys.length - 1) ? 'merge-bottom' : '';
-                                const groupFilesLength = groupFiles.length;
+                                const groupBottom = j < groupKeys.length - 1 ? 'group-bottom' : '';
 
                                 // Render each file's row, with the biological replicate and library
                                 // cells only on the first row.
                                 return groupFiles.sort((a, b) => (a.title < b.title ? -1 : 1)).map((file, i) => {
-                                    const groupBottom = i === (groupFilesLength - 1) ? 'group-bottom' : '';
+                                    // Determine whether to draw a bottom border based on whether
+                                    // this is the last file in a group (do draw) or not (don't
+                                    // draw) and it's not within the last group (don't draw).
+                                    const fileBottom = (i === groupFiles.length - 1 && groupBottom) ? 'group-bottom' : '';
```

Comment:
I found the naming a little confusing here. `groupBottom` indicates if the bottom of the group should have a bolded line (and is not within the bottom / last group) whereas I would expect it to mean that it's the bottom group (and therefore should not have a bolded line). I think it's fine to leave it as is, though.